packages/next-templates/src/app/wmebv/niet-ingelogd/stap1/page.tsx

```diff
@@ -0,0 +1,65 @@
+'use client';
+
+import {
+  UtrechtArticle,
+  UtrechtButton,
+  UtrechtButtonLink,
+  UtrechtForm,
+  UtrechtFormFieldTextarea,
+  UtrechtHeading,
+  UtrechtHeading1,
+  UtrechtHeading2,
+  UtrechtLink,
+  UtrechtPage,
+  UtrechtPageContent,
+  UtrechtParagraph,
+} from '@utrecht/web-component-library-react';
+import { ExampleHeaderWmebv } from '@/components/ExampleHeader/wmebv/ExampleHeaderWmebv';
+import { ExampleFooter } from '@/components/ExampleFooter/ExampleFooter';
+import { ExampleNavigation } from '@/components/ExampleNavigation/ExampleNavigation';
+import { UnorderedList, UnorderedListItem } from '@utrecht/component-library-react';
+import '@/app/styling/css/wmebv.css';
+
+export default function home() {
+  return (
+    <UtrechtPage>
+      <ExampleHeaderWmebv />
+      <ExampleNavigation />
+      <UtrechtPageContent className="voorbeeld-page-content-flex">
+        <UtrechtArticle className="voorbeeld-article-space ">
+          <UtrechtHeading1>Vraag aan de gemeente</UtrechtHeading1>
+          <UtrechtParagraph>Stap 1 van 4</UtrechtParagraph>
+          <UtrechtForm>
+            <UtrechtHeading2>Uw vraag</UtrechtHeading2>
+            <UtrechtFormFieldTextarea />
+          </UtrechtForm>
+          <div className="voorbeeld-bijlage-flex-container">
+            <UtrechtParagraph className="voorbeeld-paragraph-bijlage">Bijlage</UtrechtParagraph>
```

Comment:
vind persoonlijk dat je niet 2 paragraphs gebruikt voor 1 zin dat je beter er in een span gebruikt maar je kan aan robbert vragen